aggregator/deps.ts

```diff
@@ -40,17 +40,17 @@ export type {
   PublicKey,
   Signature,
   VerificationGateway,
-} from "https://esm.sh/bls-wallet-clients@0.5.3";
+} from "https://esm.sh/bls-wallet-clients@0.5.4-05c23aa";
```

Comment:
With the rebase workflow I prefer to use to consolidate commits, this will never match the merged commit unless I append it afterwards.

@voltrevo You've had some confusion in the past based on this, thoughts?

Reply:
Mmm. I understand the issue you're referring to but not the preference you're indicating above.

I think it makes sense to republish with your final changes anyway (why use a commit that represents a partially complete PR?), but let's have a chat to straighten this out.

Reply:
Just realized you can't use the final commit if you're trying to use that commit id to update the version being used. Yeah let's chat.

Reply:
@voltrevo I tried our discussed process of pinning experimental version in last commit, let me know how it looks.